src/api/crds.rs

```diff
@@ -0,0 +1,183 @@
+use crate::{
+    api::{typed::Api, Resource},
+    client::APIClient,
+};
+use inflector::{cases::pascalcase::is_pascal_case, string::pluralize::to_plural};
+use std::marker::PhantomData;
+
+/// A data equivalent of the Resource trait for for Custom Resources
+///
+/// This is the smallest amount of info we need to run the API against a CR
+/// The version, and group must be set by the user.
+pub struct CustomResource {
```

Comment:
This looks like a leftover from the days of feature-gated `k8s-openapi`.

Reply:
This is a temporary new thing until I have a fully reliable derive crate. This will be removed at some point.